proto/contlcycle/contlcycle.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/agent-payload/v5/contlcycle";
+
+package datadog.contlcycle;
+
+// EventsPayload represents the main container lifecycle event payload
+message EventsPayload {
+    enum ObjectKind {
+        Cont = 0;
+        Pod = 1;
```

Comment:
maybe we should introduce `Multi` to say that the events can have severals Kind. 
I know it will not be optimum for backend processing. but we should not close any door for futur improvement on the agent side. 

What we can document that if `multi`, they should rely on the `Kind` present in the event. Else if `container` or `pod` we certify that all events are from the same kind. WDYT?

Reply:
The current use-cases require batching by object kind, I agree that at some point in the future we could need a mixed payloads, but what would prevent us from making the change only when it becomes needed? from what I see in other teams pipelines (orchestrator for example) it's fairly easy to add fields later. 
The point is, not adding this `multi` batching now will keep things simple and pragmatic, and once new use-cases require multi kind batching in the future the change will be fairly easy in the payload.

Reply:
yes, agree to keep it simple.
my fear it that user can take wrong assumption like if it not a Pod, it is always a Container.
so instead of implementing a switch case they will just do a `if else`